### ISSUE TITLE
Test-SecretVault still fails in latest beta6 release

### ISSUE THREAD

**Issue by NJK-HOME** [JustinGrote/SecretManagement.KeePass#49]
I am still getting errors when trying to use Test-SecretVault:

```
Test-SecretVault -Name REDACTED
param : The term 'param' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

**Comment by JustinGrote** [comment]
Sorry about that, I need to add some post-compile tests to catch this kind of stuff.

**Comment by JustinGrote** [comment]
Fixed e277af94730717c3fec904be480d2e4870fff3b4 released as Beta7
